Frontend/ui-common/src/Common/Constants/Constant.purs

```diff
@@ -0,0 +1,35 @@
+
+module Common.Constants.Constant where
+
+type AndroidPolylineAnimationConfig = {
+    color :: String
+  , drawingDuration :: Int 
+  , fadingDuration :: Int
+  , delayDuration :: Int
+}
+
+type IOSPolylineAnimationConfig = {
+    color :: String 
+  , drawingPercentage :: Int 
+  , fadingPercentage :: Int 
+  , delayPercentage :: Int 
+}
+
+type PolylineAnimationConfig = {
+    android :: AndroidPolylineAnimationConfig
+  , ios :: IOSPolylineAnimationConfig
+}
+
+getPolylineAnimationConfig :: PolylineAnimationConfig 
+getPolylineAnimationConfig = {
+  android : { color : "#53BB6F"
```

Comment:
@S-PRAVINKUMAR can you please put this color in Colors.purs file.

Reply:
done